activestorage/app/models/active_storage/blob.rb

```diff
@@ -64,8 +66,16 @@ def create_after_upload!(io:, filename:, content_type: nil, metadata: nil)
     # Once the form using the direct upload is submitted, the blob can be associated with the right record using
     # the signed ID.
     def create_before_direct_upload!(filename:, byte_size:, checksum:, content_type: nil, metadata: nil)
+      raise_if_table_missing!
       create! filename: filename, byte_size: byte_size, checksum: checksum, content_type: content_type, metadata: metadata
     end
+
+    def raise_if_table_missing!
```

Comment:
I think would be good make this method private.

Reply:
Done!